utils/schema/website.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  "@context": "https://schema.org",
+  "@type": "WebSite",
+  "url": "https://vuejsconf.ir",
+  "potentialAction": {
+    "@type": "SearchAction",
+    "target": "https://vuejsconf.ir/search/?q={search_term_string}",
```

Comment:
Do we have search page?

Reply:
We do not have a search page but this is required for schema.

Reply:
But an invalid 404 route is not best idea. What if we point it to google.com/?q=

Reply:
I modified the correct search link.